perses/rjmc/topology_proposal.py

```diff
@@ -1124,9 +1210,79 @@ def propose(self, current_system, current_topology, current_metadata=None):
             msg += str(topology_proposal.new_to_old_atom_map)
             raise Exception(msg)
 
-        return topology_proposal
+        #validate the old/new system matches
+        # TODO: create more rigorous checks for this validation either in TopologyProposal or in the HybridTopologyFactory
+        #assert PolymerProposalEngine.validate_core_atoms_with_system(topology_proposal)
+
+
+        return topology_proposal, local_atom_map_stereo_sidechain, new_oemol_res, old_oemol_res
+
+    def _find_adjacent_special_atoms(self, old_topology, new_topology, mutated_residue_index):
```

Comment:
we use this function to add `N` and 'C` atoms bonded to the backbone protein residue to the alchemical region;

Reply:
maybe `_find_adjacent_backbone_atoms()` is a clearer name for the function?